indexer/api/service/service.go

```diff
@@ -0,0 +1,175 @@
+package service
+
+import (
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/ethereum-optimism/optimism/indexer/api/models"
+	"github.com/ethereum-optimism/optimism/indexer/database"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type Service interface {
+	GetDeposits(*models.QueryParams) (*database.L1BridgeDepositsResponse, error)
+	DepositResponse(*database.L1BridgeDepositsResponse) models.DepositResponse
+	GetWithdrawals(params *models.QueryParams) (*database.L2BridgeWithdrawalsResponse, error)
+	WithdrawResponse(*database.L2BridgeWithdrawalsResponse) models.WithdrawalResponse
+	GetSupplyInfo() (*models.BridgeSupplyView, error)
```

Comment:
```suggestion
	GetDeposits(*models.QueryParams) (models.DepositsResponse, error)
	GetWithdrawals(params *models.QueryParams) (models.WithdrawalsResponse, error)
	GetSupplyInfo() (*models.BridgeSupplyView, error)
```

Reply:
Can we return the model response directly rather than exporting conversion functions?